test/ios_coverage_test.sh

```diff
@@ -238,4 +297,10 @@ function test_hosted_unit_test_coverage() {
     nm -u - | grep foo || fail "Undefined 'foo' symbol not found"
 }
 
+function test_ui_test_coverage_new_runner() {
+  create_common_files
+  do_coverage ios --test_output=errors --ios_minimum_os=15.0 --experimental_use_llvm_covmap //app:test_coverage_ui_test_new_runner || fail "Should build"
+  assert_contains "CoverageApp.swift:" "test-testlogs/app/test_coverage_ui_test_new_runner/coverage.dat"
+}
```

Comment:
And then the `test-testlogs/app/test_coverage_ui_test_new_runner/coverage.dat` with your changes:
```
SF:./app/CoverageApp.swift
FN:5,$s013app_coverage_A4_lib5MyAppV4bodyQrvg
FN:6,app/CoverageApp.swift:$s013app_coverage_A4_lib5MyAppV4bodyQrvg7SwiftUI4TextVyXEfU_
FNDA:1,$s013app_coverage_A4_lib5MyAppV4bodyQrvg
FNDA:1,app/CoverageApp.swift:$s013app_coverage_A4_lib5MyAppV4bodyQrvg7SwiftUI4TextVyXEfU_
FNF:2
FNH:2
DA:5,1
DA:6,1
DA:7,1
DA:8,1
DA:9,1
BRF:0
BRH:0
LF:8
LH:8
end_of_record
SF:./app/CoverageUITest.swift
FN:4,app/CoverageUITest.swift:$s24app_coverage_ui_test_lib13PassingUiTestC0A0So15XCUIApplicationCvpfi
FN:6,$s24app_coverage_ui_test_lib13PassingUiTestC5setUpyyF
FN:11,$s24app_coverage_ui_test_lib13PassingUiTestC0D4PassyyKF
FN:12,app/CoverageUITest.swift:$s24app_coverage_ui_test_lib13PassingUiTestC0D4PassyyKFSbyKXEfu_
FNDA:1,app/CoverageUITest.swift:$s24app_coverage_ui_test_lib13PassingUiTestC0A0So15XCUIApplicationCvpfi
FNDA:1,$s24app_coverage_ui_test_lib13PassingUiTestC5setUpyyF
FNDA:1,$s24app_coverage_ui_test_lib13PassingUiTestC0D4PassyyKF
FNDA:1,app/CoverageUITest.swift:$s24app_coverage_ui_test_lib13PassingUiTestC0D4PassyyKFSbyKXEfu_
FNF:4
FNH:4
DA:4,1
DA:6,1
DA:7,1
DA:8,1
DA:9,1
DA:11,1
DA:12,1
DA:13,1
BRF:0
BRH:0
LF:9
LH:9
end_of_record
```
Seems like we'd need to change the test a bit to include validation of those `DA:` (or otherwise) lines in the coverage output vs. just the file presence.

Reply:
Good catch!